src/cascade/executor/epiviz_runner.py

```diff
@@ -523,33 +529,42 @@ def main(args):
     """
     start_time = default_timer()
     ec = make_execution_context()
+    ec.resource_monitor.start_monitor()
+    ec.resource_monitor.attach_to_process(os.getpid(), {"command": " ".join([sys.executable] + sys.argv)})
```

Comment:
Oh, you make one for this process. Nice.

Reply:
It does stuff. It's likely never the bottle neck but nice to have information about it.